heroic-component/src/main/java/com/spotify/heroic/metric/MetricCollection.java

```diff
@@ -149,6 +151,28 @@ public static MetricCollection build(
     }
 
     public static MetricCollection mergeSorted(
+        final List<MetricCollection> values
+    ) {
```

Comment:
This method only appears to be used in tests, it should probably be moved to where it is used.